nano/node/network.cpp

```diff
@@ -476,6 +477,17 @@ class network_message_visitor : public nano::message_visitor
 		{
 			node.logger.try_log (boost::str (boost::format ("Received confirm_ack message from %1% for %2% timestamp %3%") % channel->to_string () % message_a.vote->hashes_string () % std::to_string (message_a.vote->timestamp ())));
 		}
+
+		std::chrono::system_clock::time_point timestamp{ std::chrono::milliseconds{ message_a.vote->timestamp () } };
+		auto now{ std::chrono::system_clock::now () };
+		auto difference{ std::chrono::duration_cast<std::chrono::milliseconds> (now - timestamp) };
```

Comment:
Did you consider final votes? Finals votes have a special timestamp that does not represent a point in time.
The code might work anyway but it might be nice to explicitly handle it for ease of readability.